assemblies/working-in-your-data-science-ide.adoc

```diff
@@ -59,7 +59,4 @@ include::modules/troubleshooting-common-problems-in-jupyter-for-users.adoc[level
 
 include::modules/working-in-code-server.adoc[leveloffset=+1]
 
-
-ifndef::upstream[]
```

Comment:
Discussed on Slack, there is uncertainty about removing this condition. Looking at the content of the module in question, this condition should probably stay.

Reply:
Reinserted the condition